pkgs/servers/home-assistant/intents.nix

```diff
@@ -0,0 +1,62 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, setuptools
+
+# build
+, hassil
+, pyyaml
+, voluptuous
+, python
+
+# tests
+, pytest-xdist
+, pytestCheckHook
+}:
+
+buildPythonPackage {
+  pname = "home-assistant-intents";
+  version = "unstable-2023-01-12";
```

Comment:
Happy to let this PR sit for a while until they get a new version out. They use a pep517 build as of 13 hours ago, so it wasn't worth it going to the week old release.